docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -0,0 +1,14 @@
+# Permit Device To Install Applications
+
+In the menu bar, click `Tools > Tizen > Tizen Device Manager`.
+And right-click anywhere in the file explorer of the `Device Manager` and select `Permit to install applications`.
+
+![Device Manager](media/DeviceManager.png)
+
+## Troubleshooting
+
+If failed to install application to device.
+
+Check the device’s date & time.  
+If the certificate issued is before the date and time, the application cannot be installed as the certificate is considered invalid.  
+Check if you have done the Permit to install applications step.
```

Comment:
Verify if you have done the Permit to install applications step.
Is it a predefined message?
If it is, please ignore my comments.

Reply:
done